package.json

```diff
@@ -35,22 +35,22 @@
     "gulp-sourcemaps": "^1.6.0",
     "gulp-util": "^3.0.7",
     "json5": "^0.5.0",
-    "jsx-control-statements": "3.1.1",
+    "jsx-control-statements": "3.1.4",
     "less": "^2.6.1",
     "less-plugin-npm-import": "^2.1.0",
     "node-notifier": "^4.5.0",
     "node-sass": "^3.4.2",
     "raw-loader": "^0.5.1",
-    "react": "15.3.0",
-    "react-dom": "15.2.1",
+    "react": "15.3.1",
+    "react-dom": "15.3.1",
     "redbox-react": "^1.2.3",
-    "sass-loader": "^3.2.0",
+    "sass-loader": "^4.0.1",
     "string-replace-webpack-plugin": "0.0.3",
     "style-loader": "git://github.com/TerriaJS/style-loader#chromeWorkaround",
-    "svg-sprite-loader": "0.0.26",
+    "svg-sprite-loader": "0.0.27",
     "terriajs": "4.2.0",
```

Comment:
It is true that CSS changes are in Terria, however, those missing changes apply to element that do not have a specific react view (in a sense they are "global" to the Terria app), and therefore they need to be included at Aremi level. So we are be adding styles to Aremi, we are just telling Aremi to use this stylesheets from Terria.